### PR TITLE
Pushing changes upstream

### PR DESCRIPTION
**Pull request by kdreibel**
Changes from this fork so I can delete it.

* New node types
* README
* etc

@eldavido please review and merge.

### CODE REVIEW

**Comment by eldavido** [comment]
@kdreibel will review tonight, am working on another PR atm (6pmish)

File: chef_node_collection.rb [hunk 1]
```diff
@@ -35,6 +35,12 @@ def do_index_roles
       end
 
       node_role = node.run_list.to_s
+      # store the role's product name identifier if defined
+      begin
+        node_product = node[:crittercism][:tags][:product]
```
Comment (kdreibel) [inline chef_node_collection.rb]: This probably doesn't belong here ¯\_(ツ)_/¯

File: offered_instance_collection.rb [hunk 2]
```diff
@@ -21,18 +21,36 @@ def allocate!(type, az)
   def populate_on_demand
     # Populate heaps for on-demand instances
     on_demand_hourly_prices = {
-      "m1.small"    => 0.065,
-      "m1.medium"   => 0.130,
-      "m1.large"    => 0.260,
-      "m1.xlarge"   => 0.520,
-      "m3.xlarge"   => 0.550,
-      "m3.2xlarge"  => 1.100,
+      "m1.small"    => 0.047,
```
Comment (eldavido) [inline offered_instance_collection.rb]: I wonder if these could be pulled from somewhere?

**Review by eldavido — APPROVED** [review]
LGTM

I'm happy to have you guys take over the main repo for this and I can remove this, if you'd like. Let me know.

**Pull request merged**